dbt/context/common.py

```diff
@@ -24,51 +26,67 @@
 import datetime
 
 
+class RelationProxy(object):
+    def __init__(self, adapter):
+        self.quoting_config = adapter.config.quoting
+        self.relation_type = adapter.Relation
+
+    def __getattr__(self, key):
+        return getattr(self.relation_type, key)
+
+    def create(self, *args, **kwargs):
+        kwargs['quote_policy'] = dbt.utils.merge(
+            self.quoting_config,
+            kwargs.pop('quote_policy', {})
+        )
+        return self.relation_type.create(*args, **kwargs)
```

Comment:
i think i understand now -- if the primary API for creating relations is on the _adapter_ instead of the _relation_, then we can seamlessly pass in the quoting config.

i need to think a little bit more about that.

Reply:
Yup, exactly! `adapter.create_relation(...)` or something.